Library/Formula/fig.rb

```diff
@@ -56,12 +54,9 @@ def install
       r.stage { system "python", "setup.py", "install", "--prefix=#{libexec}" }
     end
 
-    system "python", "setup.py", "install", "--prefix=#{prefix}"
-
-    rm "#{lib}/python2.7/site-packages/site.py"
-    rm "#{lib}/python2.7/site-packages/easy-install.pth"
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
```

Comment:
If you could rewrite all setup.py calls to look like the ones here, that would be great: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md#example